src/pytest_mock_resources/fixture/database/generic.py

```diff
@@ -10,10 +19,21 @@ def __init__(self, drivername, host, port, database, username, password):
         self.username = username
         self.password = password
 
+    def __iter__(self):
+        for item in self.__dict__:
+            yield (item, self[item])
+
+    def __getitem__(self, item):
+        return getattr(self, item)
```

Comment:
Do we want to expose methods via `__getitem__`?

![image](https://user-images.githubusercontent.com/4128006/78305733-a87dd700-750f-11ea-8ccb-ed39295f5145.png)

Reply:
I suppose not, eh? just the dicty bits?

Reply:
I'd think we only want the dicty bits.